Library/Formula/graphene.rb

```diff
@@ -0,0 +1,43 @@
+class Graphene < Formula
+  desc "Thin layer of graphic data types"
+  homepage "https://ebassi.github.io/graphene/"
+  url "https://download.gnome.org/sources/graphene/1.2/graphene-1.2.4.tar.xz"
+  sha256 "50791cce509bc67ffc72dad8db29c46f284fd99e1784753e612702179d1aedef"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
```

Comment:
Looks like they added a fix & released a new version. If that works, feel free to update, add the `make check` bit and we should be golden.